File: practice/multiple.cpp
#include <iostream>

/*************************************************************/
bool is_divisible(int z, int n) {
    return z % n == 0;
}

void print_datum(int z, int i, int M) {
    if (i > 0) {
        std::cout << ",";
    }

    std::cout << z;
}

/*************************************************************/
int main(int argc, char* argv[]) {
    int M, N;
    int i = 0;

    std::cin >> M;
    std::cin >> N;

    for (int c = M; c <= N; c ++) {
        if (is_divisible(c, 7) && !is_divisible(c, 5)) {
            print_datum(c, i, M);
            i += 1;
        }
    }

    std::cout << std::endl;

    return 0;
}

File: practice/multiple.py
#!/usr/bin/env python3

###############################################################
def is_divisible(m, n):
    return (m % n == 0)

def print_datum(z, i, M):
    if i > 0:
        print(",", end = "")

    print(z, end = "")

###############################################################
def main():
    M = int(input())
    N = int(input())

    i = 0
    for z in range(M, N+1):
        if is_divisible(z, 7) and not is_divisible(z, 5):
            print_datum(z, i, M)
            i += 1
    
    print()



###############################################################
if __name__ == "__main__":
    main()

